alerts/google-quotas/exceeded-quota.v1.json

```diff
@@ -0,0 +1,27 @@
+{
+  "displayName": "Quota Exceeded Alert Policy",
```

Comment:
we suggest this value to match the display name in metadata so users won't get confused when seeing the list of created alert policies.

Reply:
Done.